packages/@aws-cdk/aws-synthetics/test/canary.test.ts

```diff
@@ -44,7 +44,7 @@ test('Canary can have generated name', () => {
       handler: 'index.handler',
       code: synthetics.Code.fromInline('/* Synthetics handler code */'),
     }),
-    runtime: synthetics.Runtime.SYNTHETICS_NODEJS_PUPPETEER_3_0,
+    runtime: synthetics.Runtime.SYNTHETICS_NODEJS_PUPPETEER_3_1,
```

Comment:
Why was this change if it has no effect on the generated, tested resource?

Reply:
I was following the precedence set by the [previous update](https://github.com/aws/aws-cdk/commit/6aac3b6a9bb1586ee16e7a85ca657b544d0f8304#diff-bc3779d8bebfbdd6b6484163cc900f2c772dcd10a6289f059740ef9663fa2b00). Let me know if I should revert this change.

Reply:
If this value is not tested (not included in the "expect" part), there is no point in changing it.